src/main/kotlin/uk/gov/gdx/datashare/service/EventDataService.kt

```diff
@@ -125,14 +126,21 @@ class EventDataService(
     ).increment()
     meterRegistry.timer("DATA_PROCESSING.TimeFromCreationToDeletion")
       .record(Duration.between(event.whenCreated, dateTimeHandler.now()).abs())
-    return mapEventNotification(event, consumerSubscription.eventType, false)
+    return mapEventNotification(event, consumerSubscription, false)
   }
 
-  private fun mapEventNotification(event: EventData, eventType: String, includeData: Boolean): EventNotification {
+  private fun mapEventNotification(
+    event: EventData,
+    subscription: ConsumerSubscription,
+    includeData: Boolean = false,
+    callbackEvent: Boolean = false,
+  ): EventNotification {
     return EventNotification(
       eventId = event.id,
-      eventType = eventType,
+      eventType = subscription.eventType,
       sourceId = event.dataId,
+      dataIncluded = if (!callbackEvent) includeData else null,
```

Comment:
Indicates that data not / is included and which fields would be returned. These fields are only returned for `GET /events` polling not the `GET /events/{id}`